deployment/docker-compose/docker-compose.speech.yml

```diff
@@ -7,6 +7,7 @@ services:
       - ./.gcp_credentials.json:/app/credentials.json
     env_file:
       - .litellm_proxy.env
+
```

Comment:
btw, I don't think you need the `ports: - "8001:8001"` bit since we're accessing it using `speech_service:8001` as the URL - no need to expose the port through the host machine (which is what this does) since it's accessed from within the docker network